fdbserver/VersionedBTree.actor.cpp

```diff
@@ -6310,9 +6536,9 @@ class VersionedBTree {
 			});
 		}
 
-		Future<Void> pushPage(BTreePageIDRef id) {
+		Future<Void> pushPage(PagerEventReasons r, unsigned int l, BTreePageIDRef id) {
```

Comment:
This `pushPage()` is called for the root node, so it doesn't need to accept level.  The root level approximation mentioned before can just be moved into here, `btree->m_pHeader->height`.